### PR TITLE
add nginx container

### PR DESCRIPTION
**Pull request by sos-zone**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -80,13 +80,21 @@ services:
           sleep $OSM_CHECK_TIMEOUT
         done
 
+  nginx:
+    image: nginx
+    volumes:
+      - ./nginx.conf:/tmp/nginx.conf
+    ports:
+      - "8080:80"
```
Comment (tbicr) [inline docker-compose.yml]: can we pass `80` port in env?

File: nginx.conf [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+    listen 80;
+    server_name osm.local;
+    charset utf-8;
+    error_log /var/log/nginx/error.log;
+
+    location / {
+        proxy_pass http://web:8080;
+    }
+}
```
Comment (tbicr) [inline nginx.conf]: look at config https://github.com/tbicr/osm-validator/blob/master/config/nginx.conf

File: docker-compose.yml (hunk 1 above)
Comment (sos-zone) [reply docker-compose.yml]: fixed

File: nginx.conf (hunk 2 above)
Comment (sos-zone) [reply nginx.conf]: fixed

**Pull request merged**